climada/util/coordinates.py

```diff
@@ -1945,8 +1964,8 @@ def subraster_from_bounds(transform, bounds):
     return dst_transform, dst_shape
 
 def align_raster_data(source, src_crs, src_transform, dst_crs=None, dst_resolution=None,
-                      dst_bounds=None, global_origin=(-180, 90), resampling=None, conserve=None,
-                      **kwargs):
+                      dst_bounds=None, global_origin=(-180, 90), resampling="nearest",
```

Comment:
Not related to the code review:
omg I need this method and almost wrote something myself when I couldn't find it :D

Reply:
This goes back to @sameberenz's work on the LitPop refactoring, and yeah, it's quite useful. :)